target/scripts/startup/setup.d/dmarc_dkim_spf.sh

```diff
@@ -84,3 +84,21 @@ function _setup_opendmarc
     _log 'debug' 'Disabling OpenDMARC'
   fi
 }
+
+# Configures the SPF check inside Postfix's configuration via policyd-spf. When
+# using Rspamd, you will likely want to turn that off.
+function _setup_policyd_spf
+{
+  if [[ ${ENABLE_POLICYD_SPF} -eq 1 ]]
+  then
+    _log 'debug' 'Configuring policyd-spf'
+    cat >>/etc/postfix/master.cf <<EOF
+
+policyd-spf    unix  -       n       n       -       0       spawn
+    user=policyd-spf argv=/usr/bin/policyd-spf
+EOF
+  else
+    _log 'debug' 'Disabling policyd-spf'
+    sedfile -i -E 's|check_policy_service unix:private/policyd-spf, ||g' /etc/postfix/main.cf
+  fi
```

Comment:
Wouldn't it make sense to include this in the enable logic above?

```
# From (but only for smtpd_recipient_restrictions):
# https://github.com/docker-mailserver/docker-mailserver/blob/bbe1d2da3117264c484a87c75813b0d3da71161a/target/postfix/main.cf#L51
reject_unauth_destination, reject_unauth_pipelining

# To (sed value inbetween):
reject_unauth_destination, check_policy_service unix:private/policyd-spf, reject_unauth_pipelining
```

---

Bit more work to support, I'm not going to block since this snippet will be removed before the end of the year I assume.

Reply:
I actually thought the same, but then I think it is easier with my initial way. You're right though: this should be removed by the end of the year :)